Jenkinsfile_CNP

```diff
@@ -0,0 +1,16 @@
+#!groovy
+properties([
+        [$class: 'GithubProjectProperty', projectUrlStr: 'https://github.com/hmcts/ia-case-api.git'],
+        pipelineTriggers([[$class: 'GitHubPushTrigger']])
+])
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "ia"
+def component = "case-api"
+
+withPipeline(type, product, component) {
```

Comment:
Take it you don't need to point at Azure Vault at this stage?

Reply:
Yeah we are not setup in CNP so rather wait until somebody says we have a vault.